src/raycast/camera.h

```diff
@@ -7,12 +7,28 @@
 class Camera {
 private:
   glm::vec3 mOrigin, mDirection;
+  float mFOV;
+  glm::vec3 mUp;
+  glm::vec3 mRight;
+
+private:
+  void recalculateOrientation();
 
 public:
-  Camera(glm::vec3 &o, glm::vec3 &d);
-  Ray generate_ray(float x, float y);
+  Camera();
+  Camera(glm::vec3 &o, glm::vec3 &d, float fov);
```

Comment:
make this fov=45.0f, then the test cases have a default value. and we don't need `Camera(glm::vec3 &o, glm::vec3 &d);` constructor.

Reply:
we also then don't need to specify 45.0f in the test cases.